Orange/widgets/evaluate/owtestandscore.py

```diff
@@ -383,25 +381,29 @@ def set_train_data(self, data):
         self.cancel()
         self.Information.data_sampled.clear()
         self.Error.train_data_empty.clear()
-        self.Error.class_required.clear()
-        self.Error.too_many_classes.clear()
-        self.Error.no_class_values.clear()
-        self.Error.only_one_class_var_value.clear()
+        self.Error.data_error.clear()
```

Comment:
Maybe you could rename `data_error` to `train_data_error` and use it for `train_data_empty` as well.